packages/web/docs/src/components/landing-page.tsx

```diff
@@ -27,6 +27,8 @@ import { TeamSection } from './team-section';
 export function IndexPage(): ReactElement {
   return (
     <Tooltip.Provider>
+      {/* .nextra-sidebar-container styles here are hacks. the Sidebar component should be pulled out customized and swapped */}
```

Comment:
The implementation here is _questionable_, but pulling out the Sidebar from `nextra-theme-docs` to `@theguild/components` and changing its styles would be a little bit more effort.

I'm sorry, forgive me, and let's refactor it later?

Reply:
Okay, I pulled it to a CSS file, as I'm adding a .light class to the parent of the sidebar on mount.